packages/flutter/lib/src/widgets/sliver_floating_header.dart

```diff
@@ -13,6 +13,26 @@ import 'scroll_position.dart';
 import 'scrollable.dart';
 import 'ticker_provider.dart';
 
+/// Specifies how a partially visible [SliverFloatingHeader] animates
+/// into a view when a user scroll gesture ends.
+///
+/// During a user scroll gesture the header and the rest of the scrollable
+/// content move in sync. If the header is partially visible when the
+/// scroll gesture ends, [SliverFloatingHeader.snapMode] specifies if
+/// the header should [FloatingHeaderSnapMode.overlay] the scrollable's
+/// content as it expands untill it's completely visible, or if the
```

Comment:
```suggestion
/// content as it expands until it's completely visible, or if the
```

Reply:
Thanks for all the corrections! I've fixed them.